src/scripts/ovale_rogue_spells.ts

```diff
@@ -558,14 +558,17 @@ Define(perforated_veins_conduit 248)
   SpellAddTargetDebuff(exsanguinate rupture_exsanguinated add=1 enabled=(targetdebuffpresent(rupture))) # TODO if_target_debuff is not implemented
   SpellAddTargetDebuff(exsanguinate garrote_exsanguinated add=1 enabled=(targetdebuffpresent(garrote)))
   
-  SpellDamageBuff(garrote 115191 set=1.8 enabled=(hastalent(subterfuge_talent)))
-  SpellDamageBuff(garrote subterfuge set=1.8 enabled=(hastalent(subterfuge_talent)))
-  SpellDamageBuff(garrote 11327 set=1.8 enabled=(hastalent(subterfuge_talent)))
+  SpellDamageBuff(garrote 115191 set=1.8 enabled=(specialization(assassination) and hastalent(subterfuge_talent)))
+  SpellDamageBuff(garrote subterfuge set=1.8 enabled=(specialization(assassination) and hastalent(subterfuge_talent)))
+  SpellDamageBuff(garrote 11327 set=1.8 enabled=(specialization(assassination) and hastalent(subterfuge_talent)))
 
-  # Fix broken spell definitions inherited from Simulationcraft.
-  Define(rupture_debuff 1943)
-  Define(sepsis 328305)
+# Fix broken spell definitions inherited from Simulationcraft.
+Define(rupture_debuff 1943)
```

Comment:
Manual fix to the identifiers should be done in `importsimc.ts`

Reply:
I moved this to fixes.ts 
Additionally, the test on the specialization should not be necessary anymore if there is already a test on the talent.

Reply:
The specialization tests for the Garrote debuff are needed since every rogue spec has the Subterfuge talent, but the SpellDamageBuff() lines only apply to Assassination as it's a spec-specific effect.